### PR TITLE
[incubator/kube-downscaler] - Fixes and improvements

### PR DESCRIPTION
**Pull request by Lord-Y**
Signed-off-by: Lord-Y <Lord-Y@users.noreply.github.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fix deployment
  - fix discover of resources
  - Add extraArgs

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by Lord-Y** [comment]
@zanhsieh I finally be able to made it work :)

**Comment by Lord-Y** [comment]
@davidkarlsen @prydonius can you make the review please and merge if it's okay?

**Comment by zanhsieh** [comment]
@Lord-Y 
You might find them on kubernetes slack channel (@prydonius yes, but I cannot find @davidkarlsen ).

File: incubator/kube-downscaler/templates/deployment.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ metadata:
     helm.sh/chart: {{ include "kube-downscaler.chart" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app: {{ include "kube-downscaler.name" . }}
```
Comment (prydonius) [inline incubator/kube-downscaler/templates/deployment.yaml]: why not just use app.kubernetes.io/name?

**Review by prydonius — COMMENTED** [review]
mostly lgtm, just have a question about adding the app label

File: incubator/kube-downscaler/templates/deployment.yaml (hunk 1 above)
Comment (Lord-Y) [reply incubator/kube-downscaler/templates/deployment.yaml]: @prydonius it's common to add this as it's shorter to type. I saw it in lots of charts

Comment (prydonius) [reply incubator/kube-downscaler/templates/deployment.yaml]: It's in a lot of existing charts for historical reasons (before the [recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/)), but if you're already using `app.kubernetes.io/name` I don't really see the need to add this, apart from it being faster to type. I also can't see any charts that use both `app.kubernetes.io/name` and `app` today.

**Review by prydonius — APPROVED** [review]
/lgtm

**Pull request merged**